package.json

```
@@ -31,7 +31,8 @@
     "clean": "cargo clean && rm -rf bin-package && rm -rf build",
     "test:node": "jest",
     "test:rust": "cargo test",
-    "rust:fmt": "cargo fmt --all -- --check"
+    "rust:fmt": "cargo fmt --all -- --check",
+    "rust:clippy": "cargo clippy -- -D warnings"
```

There are still 4 Clippy warnings in `smt.rs` and 6 warnings in `state_writer.rs`. In both files are Clippy warnings in tests.

In order to also check tests and non-default crate features we need to use:

```suggestion
    "rust:clippy": "cargo clippy --all-targets --all-features -- -D warnings"
```

---

Thank you for your good comments, it is done.